daft/delta_lake/delta_lake_scan.py

```diff
@@ -165,68 +159,3 @@ def can_absorb_limit(self) -> bool:
 
     def can_absorb_select(self) -> bool:
         return True
-
-
-def _storage_config_to_storage_options(storage_config: StorageConfig, table_uri: str) -> dict[str, str] | None:
```

Comment:
I realized that this stuff is shared between Python libraries that leverage the `object_store` crate, so I pulled it out to a shared utility that is now used for both DeltaLake and Lance